src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_css.md

```diff
@@ -86,19 +86,14 @@ For grouping style rules in certain media queries the `.media-width()` mixin use
 }
 
 //
-//  Tablet (for the front-end)
+//  Tablet
 //  _____________________________________________
 
-@media only screen and (max-width: @screen__m) {
-    // your code
-}
-
-//
-//  Tablets (for the back-end)
-//  _____________________________________________
-
-.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__l) {
-    // your code
+// This will add styles for tablet devices. When use native media-queries we recommended wrap your media-queries with media-width magento mixins or media-target
```

Comment:
```suggestion
// This will add styles for tablet devices. When using native media-queries, we recommended wrapping your media-queries with media-width Magento mixins or media-target
```